src/main/java/com/thevoxelbox/voxelsniper/brush/type/stencil/StencilBrush.java

```diff
@@ -49,39 +51,60 @@
     @Override
```

Comment:
Is this still somewhat of used?
![](https://i.imgur.com/rvFaEky.png)
Seems like a weird way to restrict a command. Nor have I seen it somewhere else yet.

Reply:
Not at all, let's remove this note.